apps/src/templates/HeaderBanner.jsx

```diff
@@ -7,87 +7,56 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import color from '../util/color';
 import {connect} from 'react-redux';
+import styleConstants from '@cdo/apps/styleConstants';
 
 const styles = {
-  headerBanner: {
-    height: 260,
-    maxWidth: '60%',
-    marginTop: 25
+  headerBannerContainer: {
+    minHeight: 260,
+    maxWidth: styleConstants['content-width']
   },
-  headerBannerResponsive: {
-    marginBottom: 61,
-    maxWidth: '60%',
-    marginTop: 25
-  },
-  headerBannerShort: {
-    height: 140,
-    maxWidth: '60%',
-    marginTop: 25
-  },
-  headerBannerShortResponsive: {
-    marginBottom: 61,
-    maxWidth: '60%',
-    marginTop: 25
+  headerBannerContainerShort: {
+    minHeight: 140,
+    maxWidth: styleConstants['content-width']
   },
   bannerHeading: {
     fontFamily: '"Gotham 7r", sans-serif',
     color: color.white,
     fontSize: 32,
-    marginBottom: 10,
     lineHeight: '40px'
   },
-  bannerHeadingResponsive: {
-    fontFamily: '"Gotham 7r", sans-serif',
-    color: color.white,
-    fontSize: 32,
-    marginBottom: 10,
-    lineHeight: '40px',
-    height: 240
-  },
   bannerHeadingShort: {
     fontFamily: '"Gotham 7r", sans-serif',
     color: color.white,
     fontSize: 32,
-    marginBottom: 10,
-    lineHeight: '40px',
-    marginTop: -20
-  },
-  bannerHeadingShortResponsive: {
-    fontFamily: '"Gotham 7r", sans-serif',
-    color: color.white,
-    fontSize: 32,
-    marginBottom: 10,
-    lineHeight: '40px',
-    marginTop: -20,
-    height: 120
+    lineHeight: '40px'
   },
   bannerSubHeading: {
     fontFamily: '"Gotham 4r", sans-serif',
     color: color.white,
     fontSize: 16,
     lineHeight: '21px',
-    marginBottom: 10
+    marginTop: '16px'
```

Comment:
I think these can be `16` rather than `'16px'`?

Reply:
fixed. I checked `lineHeight` as well but it doesn't auto convert to pixels.

Reply:
Yes, good point.  `line-height` in CSS defaults to being a multiplier of the `font-size`.